.github/workflows/main.yml

```diff
@@ -0,0 +1,19 @@
+name: Check PR title and project linting
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - edited
+      - synchronize
+
+jobs:
+  lint:
+    if: "!contains(github.event.pull_request.head.ref, 'renovate')"
+    runs-on: ubuntu-latest
+    steps:
+    - uses: deepakputhraya/action-pr-title@master
+      with:
+        regex: '^(CIV-[0-9]{3,4} [a-zA-Z0-9._\- ]+)|(\[Snyk\].+)$' # Regex the title should match.
+        allowed_prefixes: 'CIV-,[Snyk]' # title should start with the given prefix
```

Comment:
what if we don't have a Jira for some quick task? i.e. fix a build etc.

Reply:
Everything should have a task so it can be traced back. A hack round it would be add a Prefix of CIV-9999